tests/io_perfomance/image/README.md

```diff
@@ -0,0 +1,46 @@
+# FIO autotests container
```

Comment:
This looks like exactly the same README as the one in image-volume. Can we please kill one of those?